planning/mission_planner/README.md

```diff
@@ -25,13 +25,15 @@ In current Autoware.universe, only Lanelet2 map format is supported.
 
 ### Services
 
-| Name                                             | Type                                      | Description                                 |
-| ------------------------------------------------ | ----------------------------------------- | ------------------------------------------- |
-| `/planning/mission_planning/clear_route`         | autoware_adapi_v1_msgs/srv/ClearRoute     | route clear request                         |
-| `/planning/mission_planning/set_route_points`    | autoware_adapi_v1_msgs/srv/SetRoutePoints | route request with pose waypoints           |
-| `/planning/mission_planning/set_route`           | autoware_planning_msgs/srv/SetRoute       | route request with lanelet waypoints        |
-| `/planning/mission_planning/change_route_points` | autoware_adapi_v1_msgs/srv/SetRoutePoints | route change request with pose waypoints    |
-| `/planning/mission_planning/change_route`        | autoware_planning_msgs/srv/SetRoute       | route change request with lanelet waypoints |
+| Name                                             | Type                                              | Description                                 |
+| ------------------------------------------------ | ------------------------------------------------- | ------------------------------------------- |
+| `/planning/routing/clear_route`                  | autoware_adapi_v1_msgs/srv/ClearRoute             | route clear request                         |
+| `/planning/routing/set_route_points`             | autoware_adapi_v1_msgs/srv/SetRoutePoints         | route request with pose waypoints           |
+| `/planning/routing/set_route`                    | autoware_planning_msgs/srv/SetRoute               | route request with HAD map format           |
+| `/planning/mission_planning/change_route_points` | autoware_adapi_v1_msgs/srv/SetRoutePoints         | route change request with pose waypoints    |
+| `/planning/mission_planning/change_route`        | autoware_planning_msgs/srv/SetRoute               | route change request with lanelet waypoints |
+| `~/srv/set_mrm_goal`                             | autoware_planning_msgs/srv/SetPoseWithUuidStamped | set emergency route                         |
+| `~/srv/clear_mrm_goal`                           | std_srvs/srv/Trigger                              | clear emergency route                       |
```

Comment:
mrm_route

https://github.com/autowarefoundation/autoware.universe/pull/3299

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/3356/commits/6bfaedf8b6f75add44aceece2d86de7cd082c15a